src/components/Editor/TextEditor.tsx

```diff
@@ -13,11 +13,11 @@ const TextDiv = styled.div`
   background: #3f3d56;
   display: flex;
   width: inherit;
-  height:inherit;
+  height: inherit;
   flex-direction: column;
   border-radius: 15px;
```

Comment:
is it possible to keep using em instead of px

Reply:
for border radius, px should be fine but I used em for all the other sizings